heartbeat/podman-etcd

```diff
@@ -1513,6 +1528,10 @@ podman_validate()
 		exit $OCF_ERR_CONFIGURED
 	fi
 
+	if [ "$OCF_RESKEY_oom" -lt -1000 ] || [ "$OCF_RESKEY_oom" -gt 1000 ]; then
+		ocf_exit_reason "'oom' value ${OCF_RESKEY_oom} is out of range [-1000:1000]"
```

Comment:
You should `exit $OCF_ERR_CONFIGURED` here.

Reply:
oh, sure :facepalm: